core/packages/test/deployment/goerli-cd9e04958cb72e3fb1026c80b0e6ed032c9e83a6-1686329993.json

```diff
@@ -0,0 +1,3925 @@
+{
```

Comment:
Should we add this file to gitignore, or is it being used somewhere?

Reply:
Yes, the deployment file here is just for placeholder and I can delete it if nessessary.  But IMO saving deployment history somewhere could be useful for production usage when we would like to debug on chain or upgrade from previous. The naming rule here is `${network}-${commitId}-${timestamp}`.